test/provisioner/provision.go

```diff
@@ -62,6 +70,42 @@ type InstallOverlay interface {
 	Edit(ctx context.Context, cfg *envconf.Config, properties map[string]string) error
 }
 
+func NewKeyBrokerService(provider string) (*KeyBrokerService, error) {
+	// Clone kbs repo
+	repoURL := "https://github.com/confidential-containers/kbs"
+	cmd := exec.Command("git", "clone", repoURL)
```

Comment:
We need to also pin down the KBS version that we will pull for reliable tests.

Reply:
Yes we can decide on that. @mkulke  any suggestions which one we should use ?

Reply:
the commit from the current should be fine